airflow/www/views.py

```diff
@@ -82,6 +82,7 @@
 
 PAGE_SIZE = conf.getint('webserver', 'page_size')
 FILTER_TAGS_COOKIE = 'tags_filter'
+FILTER_STATUS_COOKIE = 'status_filter'
```

Comment:
```suggestion
FILTER_STATUS_COOKIE = 'dag_status_filter'
```

is a slightly more descriptive cookie name.

Reply:
I was just continuing the pattern of the cookie above it, but understand the desire. Should I go ahead and give that one a `dag_` prefix too?

Reply:
Updated (just the status cookie).